pkg/verifier/notation/truststore.go

```diff
@@ -72,13 +72,13 @@ func (s *trustStore) getCertificatesInternal(ctx context.Context, storeType trus
 			logger.GetLogger(ctx, logOpt).Debugf("truststore getting certStore %v", certStore)
 			certMap, kmpErr := keymanagementprovider.GetCertificatesFromMap(ctx, certStore)
 			if kmpErr != nil {
-				logger.GetLogger(ctx, logOpt).Warnf("unable to fetch certificates for Key Management Provider %+v: %v", certStore, kmpErr)
+				logger.GetLogger(ctx, logOpt).Infof("unable to fetch certificates for Key Management Provider %+v: %v", certStore, kmpErr)
 			}
 			result := keymanagementprovider.FlattenKMPMap(certMap)
 			var certStoreErr error
 			// notation verifier does not consider specific named/versioned certificates within a key management provider resource
 			if len(result) == 0 {
-				logger.GetLogger(ctx, logOpt).Warnf("no certificate fetched for Key Management Provider %+v", certStore)
+				logger.GetLogger(ctx, logOpt).Infof("no certificate fetched for Key Management Provider %+v", certStore)
```

Comment:
Hi Binbin, is this a duplicate log ? "no certificate fetched for Key Management Provider %+v" , since we already have the above "unable to fetch certificates for Key Management Provider %+v: %v"

Reply:
the previous log was printed when an error happened, the second one is always printed if no certs found.